src/tests/unit/userProfileSettings.test.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import cookieParser from 'cookie-parser';
+import bcrypt from 'bcrypt';
+import { describe, it } from 'mocha';
+import app from '../../index.js';
+import User from '../../models/user.model.js';
+import sequelize from '../../database/config/db.js';
+
+chai.should();
+chai.use(chaiHttp);
+chai.use(cookieParser);
+const { expect } = chai;
+
+const { hash } = bcrypt;
+
+describe('PUT /settings', () => {
+  let token = '';
+  let userId = '';
+  let idx;
+  before(async () => {
+    sequelize.sync();
```

Comment:
This is wrong. We won't be syncing the db from models. I need file migrations to create the table before all tests